app/controllers/users_controller.rb

```diff
@@ -843,15 +843,15 @@ def search_users
     topic_id = topic_id.to_i if topic_id
     topic_allowed_users = params[:topic_allowed_users] || false
 
-    if params[:group].present?
-      @group = Group.find_by(name: params[:group])
+    if params[:groups].present?
```

Comment:
This looks like a breaking API change to me. We need to ensure that if people are using the old `group` parameter that it continues to work.

Reply:
Yes, it's a breaking change. Looks like only `discourse-assign` is using this option though (searched in core and the `all-the-plugins` repo).